.mergify.yml

```diff
@@ -0,0 +1,56 @@
+pull_request_rules:
+    - name: automatic merge on all ci success and review
```

Comment:
I'm not sure we want to do this by default. Probably should be an agenda item.

Reply:
I'd be more willing if the tag "DO NOT MERGE" was applied by default.

Reply:
Assuming all necessary reviews are "approve" and the CI passes, then why shouldn't we merge? I assume an author would add the "DO NOT MERGE" label if they don't want it to be merged...

Reply:
I just think in practice people (me, at least) don't apply that label, because people don't expect others will click the green button for them. Additionally, some people (me) give conditional approval but still expect changes. 

Of course, these criticisms can be addressed with a process change -- we should talk about those at a chipyard meeting.

Reply:
The `change-requested` flag requires that only reviews without changes requested will go through (probably need to double check though). Another option is to add a `MERGE` label which tells the "bot" to merge when ready.